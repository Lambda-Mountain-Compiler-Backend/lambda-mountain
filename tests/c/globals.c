
int global_x = 5, global_y, global_z = 4;
int main(){}
