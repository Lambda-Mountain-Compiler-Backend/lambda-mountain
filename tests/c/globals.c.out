int global_x = 5;
int global_y;
int global_z = 4;
int main(){}
